app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -47,13 +50,44 @@ class NewsSideBar extends ConsumerWidget {
     );
 
     return Column(
+      mainAxisAlignment: MainAxisAlignment.end,
       children: <Widget>[
-        const Spacer(),
-        LikeButton(
-          likeCount: news.likesCount().toString(),
-          style: style,
-          color: fgColor,
-          index: index,
+        reactionManager.maybeWhen(
+          data: (manager) {
+            return SizedBox(
+              height: 75,
+              width: 75,
+              child: LikeButton(
+                isLiked: reactionStatus.valueOrNull ?? false,
+                likeCount: reactions.valueOrNull != null
+                    ? reactions.valueOrNull!.length.toString()
+                    : '0',
+                style: style,
+                color: fgColor,
+                index: index,
+                onTap: () async {
+                  final client = ref.read(alwaysClientProvider);
+                  final status = await manager.myStatus();
+                  debugPrint('STATUS: $status');
+                  if (!status) {
+                    final eventId = await manager.sendReaction(
+                      news.eventId().toString(),
+                      '\u{dd25}',
+                    );
```

Comment:
I'd suggest a rust-method "sendLike" and "sendUnlike" which deal with the details internally in rust. Also, again, this API should probably not require `news.eventId()` at all, considering that it is derived from `news` and thus should implicitly know that context ...

Reply:
When we want `sendLike` and `sendUnlike`, we should use `U+1F44D` and `U+1F44E` as symbol?

Reply:
And we should keep both `reaction_count()` and `like_count()`?

Reply:
we are internally just falling back [the ruma `Annotation`](https://docs.ruma.io/ruma/events/relation/struct.Annotation.html), right? Which just uses some `String`. So, yeah, let's agree that we interpret the unicode code points for the [Red Hearts emoji](https://emojipedia.org/red-heart) (`U+2764`&`U+FE0F` together) as our Acter-Like. Makes it somewhat compatible with existing Matrix and we can expand the counter of what else we consider a like later as well.

I recommend to put that as a static str somewhere then reference in the rust-code-base @bitfriend .

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/4a9a1127db3f96f86a98f464c2fe55b4337c3b4f